application/modules/home/views/side_ci.php

```diff
@@ -0,0 +1,5 @@
+<h3 class="box-title"><i class="fa fa-building"></i>
```

Comment:
__[code_sniffer]__
* File "side_ci.php" should be saved with Unicode (UTF-8) encoding, but it did not successfully pass the W3C test.

<!-- Comment by [SideCI](https://sideci.com) -->